tutorials/3d/introduction_to_3d.rst

```diff
@@ -24,6 +24,10 @@ Node3D node
 Following this reasoning, the 3D engine uses the :ref:`Node3D <class_Node3D>`
 node for everything 3D.
 
+.. note:: Be aware that there was a renaming of "Spatial" Nodes to
+   "Node3D" starting Godot 4. Any Godot 3.x references about "Spatial"
+   Nodes refer to "Node3D" in Godot 4.
```

Comment:
```suggestion
.. note:: Be aware that "Spatial" Nodes are now called "Node3D"
   starting with Godot 4.. Any Godot 3.x references to "Spatial"
   Nodes refer to "Node3D" in Godot 4.
```

Reply:
Is the double dot after `with Godot 4` intentional?

Reply:
I doubt it :)